### ISSUE TITLE
reach_material : PRV au lieu de GUP

### ISSUE THREAD

**Issue by varrieta** [QGEP/QGEP#616]
Dans la table qgep_vl.reach_material, la valeur value_fr = matiere_synthetique_polyester_GUP correspond-elle au PRV ?

Si oui, GUP ne devrait-il pas être remplacé par PRV ?

**Comment by sjib** [comment]
These values come from the SIA405 norm. Can you check the french version? 
If it is the same there we have to hand in a change request to SIA / VSA to correct this. 
What does PRV mean in French (abreviation for what?)

**Comment by varrieta** [comment]
In the French version it is also "matiere_synthetique_polyester_GUP"

PRV mean in french :  polyester renforcé de fibre de verre

And the traduction for GUP seems to be : GUP (glass fibre reinforced unsaturated polyester)

**Comment by sjib** [comment]
So do I understand right that the French translation should be 
"matiere_synthetique_PRV" or
 "matiere_synthetique_polyester_renforce_de_fibre_verre" to be correct? 
Or "matiere_synthetique_polyester_RV"?

**Comment by varrieta** [comment]
Yes that is corret. I would go for the first one because the abbreviation PRV is well known.

And also it goes in the same idea as "matiere_synthetique_PVC" (for example).

**Comment by sjib** [comment]
Ok, I will correct that in the new model version of the VSA Release 2020 in the French version planned for Spring 2021
